recipes/vit-pytorch/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "vit-pytorch" %}
+{% set version = "0.25.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2c9c71ddafbb390f0385ecd3ed728f3bc78678b5e89420c25f6360f6d4034b52
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - einops >=0.3
+    - torch >=1.6
```

This is `pytorch` in the conda ecosystem:
```suggestion
    - pytorch >=1.6
```

---

oh yes, thanks Ryan!